packages/playwright/bundles/babel/src/babelBundleImpl.ts

```diff
@@ -120,8 +120,15 @@ export function babelTransform(code: string, filename: string, isTypeScript: boo
 
   // Prevent reentry while requiring plugins lazily.
   isTransforming = true;
+  let options: TransformOptions;
+
   try {
-    const options = babelTransformOptions(isTypeScript, isModule, pluginsPrologue, pluginsEpilogue);
+    if (process.env.PW_TEST_BABEL_TRANSFORM_OPTIONS_FILE) {
+      const importedBabelTransformOptions = require(process.env.PW_TEST_BABEL_TRANSFORM_OPTIONS_FILE);
+      options = importedBabelTransformOptions(isTypeScript, isModule, pluginsPrologue, pluginsEpilogue);
```

Comment:
I don't think this will work if we decide to migrate to esbuild for transpilation.